app/controllers/vacancies_controller.rb

```diff
@@ -3,8 +3,7 @@ class VacanciesController < ApplicationController
 
   def index
```

Comment:
I agree, though the way I did it was the opposite way to how it might have been logically broken down in commits (i.e commits could have been rename, refactor, new feature). And once I'd done it there was no going back (well there was but it would have been a pain) - so I hope you forgive me 🙏